main.star

```diff
@@ -77,9 +77,17 @@ def run(args):
 			print("Waiting for the first finalized epoch")
 			first_cl_client = all_cl_client_contexts[0]
 			first_cl_client_id = first_cl_client.beacon_service_id
-			define_fact(service_id = first_cl_client_id, fact_name = FIRST_NODE_FINALIZATION_FACT, fact_recipe = struct(method= "GET", endpoint = "/eth/v1/beacon/states/head/finality_checkpoints", content_type = "application/json", port_id = HTTP_PORT_ID_FOR_FACT, field_extractor = ".data.finalized.epoch"))
-			finalized_epoch = wait(service_id = first_cl_client_id, fact_name = FIRST_NODE_FINALIZATION_FACT)
-			# TODO make an assertion on the finalized_epoch > 0
+			epoch_recipe = struct(
```

Comment:
can you pass a time out as this sometimes needs 32 minutes

Reply:
well actually there's some math we can do to calculate that https://github.com/kurtosis-tech/eth2-merge-kurtosis-module/blob/master/kurtosis-module/impl/module.go#L229-L262

Reply:
that sounds good